.github/workflows/setup_build_environment.yml

```diff
@@ -16,9 +16,13 @@ jobs:
         uses: actions/checkout@v3
       - name: Setup Android SDK
         uses: android-actions/setup-android@v2.0.10
-      - name: Read google-service.json
+      - name: Save google-service.json
         env:
           FIREBASE_GOOGLE_SERVICE: ${{ secrets.FIREBASE_GOOGLE_SERVICE }}
         run: echo $FIREBASE_GOOGLE_SERVICE > app/src/google-services.json
+      - name: Save service-credentials.json
+        env:
+          FIREBASE_SERVICE_CREDENTIALS: ${{ secrets.FIREBASE_SERVICE_CREDENTIALS }}
+        run: echo $FIREBASE_SERVICE_CREDENTIALS > app/src/service-credentials.json
```

Comment:
I think we should rebuild this "basic" workflow, it's a nice example how basic script/class/etc. could grow too much.  `setup_build_environment` is also used for executing ktlint and tests, where it's totally redundant. Candidate for a separate issue.

Reply:
I fully agree. I've created an issue for that: https://github.com/intive/Picover/issues/208